src/main/java/kg/jarkyn/cobspecserver/Runner.java

```diff
@@ -12,55 +12,64 @@
 
 import java.io.IOException;
 import java.net.ServerSocket;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Runner {
-    private static boolean KEEP_RUNNING = true;
-    private static int defaultPort      = 5000;
-    private static String publicResourcePath = "/Users/Jarkyn/Projects/8thLight/JAVA/cob_spec/public";
+    private static final int DEFAULT_PORT = 5000;
+    private static final boolean KEEP_RUNNING = true;
+    private static final String DOMAIN = "http://localhost";
+    private static final String PUBLIC_RESOURCE_PATH = "/Users/Jarkyn/Projects/8thLight/JAVA/cob_spec/public";
```

Comment:
I don't think `DEFAULT_PORT` and `PUBLIC_RESOURCE_PATH` should be constants. Those values happen to be the same CobSpec uses by default.
But https://github.com/8thlight/cob_spec#http-server specifies that CobSpec expects to be able to send them as arguments. 
I don't know if this is in the scope of your iteration, I would ask your mentors if you are not sure.